docs/user-guide/time-and-costs/cost-tracking/README.md

```diff
@@ -22,7 +22,9 @@ You can log unit costs on a work package within a project to keep track on certa
 
 To track unit costs to a work package select the **Log unit costs** option from the **More functions** drop down menu.
 
-![Cost-tracking_log-costs](Cost-tracking_log-costs.png)
+<video src="/Users/jankeppler/Desktop/Bildschirmaufnahme 2022-02-12 um 18.44.32.mov"></video>
```

Comment:
This video is only on your computer and it will not work in many browsers, since mov is not supported by many. Please create animated gifs.